cmake/modules/BuildJaeger.cmake

```diff
@@ -73,7 +73,8 @@ function(build_jaeger)
   else()
     set(make_cmd ${CMAKE_COMMAND} --build <BINARY_DIR> --target jaegertracing)
   endif()
-  set(install_cmd ${CMAKE_MAKE_PROGRAM} install)
+  set(install_cmd DESTDIR= ${CMAKE_MAKE_PROGRAM} install)
```

Comment:
i still don't get why / how an empty `DESTDIR` helps. could you elaborate a little bit?